### PR TITLE
Adds info on increasing restic memory limit

### PR DESCRIPTION
**Pull request by paigecalvert**
https://app.shortcut.com/replicated/story/54089/document-workaround-for-increasing-the-memory-limit-of-restic-pod

Previews:

- See the second "Replicated recommends that you..." prerequisite https://deploy-preview-496--replicated-docs.netlify.app/enterprise/snapshots-storage-destinations#prerequisites
- New "Memory Limit" troubleshooting section: https://deploy-preview-496--replicated-docs.netlify.app/enterprise/snapshots-troubleshooting-backup-restore#memory-limit-reached-on-the-restic-pod

### CODE REVIEW

File: docs/enterprise/snapshots-troubleshooting-backup-restore.md [hunk 1]
```diff
@@ -131,3 +133,23 @@ Replace `TIMEOUT_LIMIT` with a length of time for the restic Pod operation timeo
 :::note
 Application install overwrites the restic timeout that you set with the `kubectl patch deployment velero` command.
 :::
+
+### Memory Limit Reached on the Restic Pod
+
+#### Symptom
+
+The restic Pod is killed by the Linux kernel Out Off Memory (OOM) killer.
```
Comment (xavpaice) [inline docs/enterprise/snapshots-troubleshooting-backup-restore.md:141]: s/Off/Of/

**Review by xavpaice — COMMENTED** [review]
One tiny tiny nit inline, otherwise LGTM.  Thanks for doing this!

File: docs/enterprise/snapshots-troubleshooting-backup-restore.md (hunk 1 above)
Comment (paigecalvert) [reply docs/enterprise/snapshots-troubleshooting-backup-restore.md]: Thank you!

File: docs/enterprise/snapshots-storage-destinations.md [hunk 2]
```diff
@@ -1,30 +1,58 @@
 # Storage Destinations
 
-You can use compatible Velero storage providers with the Replicated snapshot feature. For more information, see [Providers](https://velero.io/docs/v1.9/supported-providers/) in the Velero documentation.
+This topic describes the supported storage destinations for backups with the Replicated snapshots feature. It includes the prerequisites for configuring storage destinations for snapshots, and lists the available fields for cloud storage destinations.
 
-The Replicated admin console has built-in support for using the following as storage destinations for snapshots:
+## Overview of Configuring Storage destinations
```
Comment (jonquil2002) [inline docs/enterprise/snapshots-storage-destinations.md:5]: s/ destinations / Destinations (make uppercase)

File: docs/enterprise/snapshots-storage-destinations.md [hunk 3]
```diff
@@ -1,30 +1,58 @@
...
-For more information, see [`velero ensure-permissions`](../reference/kots-cli-velero-ensure-permissions/) in the kots CLI documentation.
+## Prerequisites
 
-For more information about RBAC privileges for the admin console, see [Kubernetes RBAC](../vendor/packaging-rbac).
+* Velero must be installed on the cluster to use snapshots:
+   * Existing clusters: Install Velero on the cluster. See [Basic Install](https://velero.io/docs/v1.9/basic-install/) in the Velero documentation.
+   * Kubernetes installer clusters: Your application vendor can provide the Velero add-on in the Kubernetes installer cluster. If Velero is not already installed on the Kubernetes installer cluster, the snapshots configuration dialog in the admin console notifies you to install Velero before you can proceed with the configuration.
 
-## Prerequisites for Cloud Configurations
+* Replicated recommends that you increase the default memory limit for the restic Pod on the Velero deployment.
```
Comment (jonquil2002) [inline docs/enterprise/snapshots-storage-destinations.md:19]: I'm curious about why "Pod" is uppercase. Also, in the next paragraph, it's "Velero Pod" and "restic Pod". My thought is that pod should be lowercase. And should restic be uppercase, so "Restic pod"?

**Review by jonquil2002 — APPROVED** [review]
lgtm, just a few minor suggestions/question. What a huge improvement to this content! Kudos!

File: docs/enterprise/snapshots-storage-destinations.md (hunk 2 above)
Comment (paigecalvert) [reply docs/enterprise/snapshots-storage-destinations.md:5]: good catch 👍

**Review by jonquil2002 — APPROVED** [review]
lgtm

**Pull request merged**